project/Dependencies.scala

```diff
@@ -69,12 +69,13 @@ object Dependencies {
   val lz4                 = "org.lz4"                     % "lz4-java"                  % "1.7.1"
   val magnolia            = "com.propensive"             %% "magnolia"                  % "0.17.0"
   val monix               = "io.monix"                   %% "monix"                     % monixVersion
+  val monixTesting        = "io.monix"                   %% "monix-testing-scalatest"   % "0.3.0"
   val pureconfig          = "com.github.pureconfig"      %% "pureconfig"                % "0.14.0"
   val scalaLogging        = "com.typesafe.scala-logging" %% "scala-logging"             % "3.9.4"
   val scalaUri            = "io.lemonlabs"               %% "scala-uri"                 % "3.0.0"
   val scalacheck          = "org.scalacheck"             %% "scalacheck"                % "1.15.0"
   val scalacheckShapeless = "com.github.alexarchambault" %% "scalacheck-shapeless_1.15" % "1.3.0"  % "test"
-  val scalactic           = "org.scalactic"              %% "scalactic"                 % "3.2.10" % "test"
+  val scalactic           = "org.scalactic"              %% "scalactic"                 % "3.2.9" % "test"
```

Comment:
Is it fallback because of compatibility with new `monixTesting`?

Reply:
@zsluedem, yes, exactly. From librarie's page
```
5 Dependencies

com-lihaoyi/utest             0.7.10
monix/minitest                2.9.6
monix/monix                   3.4.0
scalatest/scalatest           3.2.9  <-- 
typelevel/cats-effect-testing 0.5.4
```